packages/edit-site/src/components/sidebar/global-styles-v2/components/color-picker-modal.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	CloseButton,
+	ColorPicker,
+	Heading,
+	HStack,
+	View,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useAppState } from '../state';
+
+export function ColorPickerModal() {
```

Comment:
Should we call this one `ColorPickerDropdown` or `Menu`? We are looking at something like this, design wise:

![image](https://user-images.githubusercontent.com/548849/123419939-23228e80-d5bb-11eb-85bc-e59bd3f2b170.png)